### PR TITLE
Add ubuntu image build

### PR DESCRIPTION
**Pull request by shahar-klein**

### CODE REVIEW

File: dist/images/Dockerfile.ubuntu [hunk 1]
```diff
@@ -0,0 +1,51 @@
+#
+# The standard name for this image is ovn-kube
```
Comment (rajatchopra) [inline dist/images/Dockerfile.ubuntu]: Can you change the comment to reflect what is really built finally in the Makefile? 'ovn-kube-u' that is.

File: dist/images/Dockerfile.ubuntu [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+# Notes:
+# This is for a development build where the ovn-kubernetes utilities
+# are built in this Dockerfile and included in the image (instead of the rpm)
```
Comment (rajatchopra) [inline dist/images/Dockerfile.ubuntu]: s/rpm/deb?

File: dist/images/Dockerfile.ubuntu (hunk 1 above)
Comment (shahar-klein) [reply dist/images/Dockerfile.ubuntu]: Done

File: dist/images/Dockerfile.ubuntu (hunk 2 above)
Comment (shahar-klein) [reply dist/images/Dockerfile.ubuntu]: Done

**Comment by shettyg** [comment]
I tested this and looks okay. Ubuntu 18.04 only gives OVS 2.9.0 which has some bugs. So as part of a separate commit, I will likely change the source of OVS deb packages to the one I maintain for amazon.

**Comment by shettyg** [comment]
@shahar-klein 

I realized that this patch does not have a Signed-off-by after merging it. Can you please write 
Signed-off-by: shahar klein <sklein@nvidia.com>  in the comments below?

**Comment by shahar-klein** [comment]
Signed-off-by: shahar klein sklein@nvidia.com

**Pull request merged**